pool/app/pool_common/locales/locales_en.ml

```diff
@@ -99,6 +99,7 @@ let field_to_string =
   | Operator -> "operator"
   | Operators -> "operators"
   | Overbook -> "overbook"
+  | OverriddenValue -> "overriden value"
```

Comment:
May use "overridden contact value" to be more precise for admins.

Btw, overwritten or overridden?
(https://github.com/uzh/pool/pull/87/files#diff-37b8f41068825878103c2771817f8dc19bd25ae977ee8c688eb782fdf21ff8eaL26)

Reply:
Changed it to "overridden contact answer".

Regarding overwritten / overridden: https://stackoverflow.com/questions/8651562/overwrite-or-override#:~:text=To%20%22overwrite%22%20something%20is%20to,or%20changing%20the%20thing%20overridden

According to this definition, it would be override.

Reply:
Could you change the "overwritten" to "overridden" in the project?

Reply:
Done, its the last commit